src/community_advisors/models/de.rs

```diff
@@ -64,4 +66,29 @@ mod tests {
         let data: Vec<AdvisorReviewRow> = csv_utils::load_data_from_csv(&file_path).unwrap();
         assert_eq!(data.len(), 1);
     }
+
+    impl AdvisorReviewRow {
+        pub fn dummy(score: ReviewScore) -> Self {
```

Comment:
Is this just used for testing? If thats the case, let's move it somewhere better, as a separated function or maybe a feature flag.

Reply:
It's under a test module with `#[cfg(test)]` now, is that what you mean? Otherwise I'm open to suggestions, we could even implement something like proptest's `Arbitrary`.

Reply:
yeah, under `#[cfg(test)]` is ok.